pkg/graveler/ref/manager.go

```diff
@@ -629,3 +632,33 @@ func (m *Manager) DeleteExpiredLinkAddresses(ctx context.Context, repository *gr
 	}
 	return itr.Err()
 }
+
+func (m *Manager) DeleteExpiredImports(ctx context.Context, repository *graveler.RepositoryRecord) error {
+	expiry := time.Now().Add(-ImportExpiryTime)
+	repoPartition := graveler.RepoPartition(repository)
+	key := []byte(graveler.ImportsPath(""))
+	options := kv.IteratorOptionsFrom([]byte(""))
+	itr, err := kv.NewPrimaryIterator(ctx, m.kvStoreLimited, (&graveler.ImportStatusData{}).ProtoReflect().Type(), repoPartition, key, options)
+	if err != nil {
+		return err
+	}
+	defer itr.Close()
+
+	for itr.Next() {
+		entry := itr.Entry()
+		status, ok := entry.Value.(*graveler.ImportStatusData)
+		if !ok {
+			return graveler.ErrReadingFromStore
```

Comment:
Can we add the actual dynamic type of `entry.Value` here somehow?  (Perhaps @nopcoder will know?)  Otherwise when it fails there will be no information.  If not, consider wrapping this error with the first few bytes of the binary `Value`.

Reply:
Done